packaging/examples/cruise-control/kafka-cruise-control.yaml

```diff
@@ -23,6 +23,10 @@ spec:
       min.insync.replicas: 2
       log.message.format.version: "3.0"
       inter.broker.protocol.version: "3.0"
+    resources:
+      limits:
+        cpu: 1000m
+        memory: 1Gi
```

Comment:
Why is this being added? I don't think we want any specific values in the example resources as they don't necessarily work.

Reply:
We recommend to set a limit, but our own example has no limit, this is in contrast. What about adding a comment on top of resources which says "set your own values here"?

Reply:
None of our examples have any requests or limits, because it causes problems and misunderstandings: you do not know which setting is the right for the user and which actually works in his environment. And you also do not want to suggest to anyone that `cpu: 1000m` and `memory: 1Gi` is what you recommend them to use.

Reply:
Ok, agree. Will remove it.